tests/test_data_paths.py

```diff
@@ -7,9 +7,11 @@
 
 import rasterio
 from rasterio._env import GDALDataFinder, PROJDataFinder
+from .conftest import requires_gdal_lt_3
 from rasterio.rio.main import main_group
 
 
+@requires_gdal_lt_3
```

Comment:
We still need this test with GDAL 3. It confirms that the environment patching we do for wheels distributions will work.